lib/tasks/heart_rate_data.rake

```diff
@@ -0,0 +1,9 @@
+namespace :heart_rate_data do
+  desc "Update Fitbit heart rate data for the last user"
+  task sync: :environment do
+    user = User.last
+    user.heart_rate_datas << HeartRateData.new
+
+    user.heart_rate_datas.last.update_heart_rate_data!
```

Comment:
Sorry, I missed that one! I've now moved that logic into a method on the User class